src/Mod/Arch/ArchWall.py

```diff
@@ -39,23 +39,65 @@ def QT_TRANSLATE_NOOP(ctxt,txt):
 #  \ingroup ARCH
 #  \brief The Wall object and tools
 #
-#  This module provides tools to build Wall objects.
-#  Walls are simple objects, usually vertical, obtained
-#  by giving a thickness to a base line, then extruding it
-#  vertically.
+#  This module provides tools to build Wall objects.  Walls are simple objects,
+#  usually vertical, typically obtained by giving a thickness to a base line,
+#  then extruding it vertically.
 
 __title__="FreeCAD Wall"
 __author__ = "Yorik van Havre"
 __url__ = "http://www.freecadweb.org"
 
-
-
-def makeWall(baseobj=None,length=None,width=None,height=None,align="Center",face=None,name="Wall"):
-
-    '''makeWall([obj],[length],[width],[height],[align],[face],[name]): creates a wall based on the
-    given object, which can be a sketch, a draft object, a face or a solid, or no object at
-    all, then you must provide length, width and height. Align can be "Center","Left" or "Right",
-    face can be an index number of a face in the base object to base the wall on.'''
+__doc__="""This module provides tools to build Wall objects.  Walls are simple
```

Comment:
I would add this documentation before the `\ingroup ARCH` block, which is meant for Doxygen.

Reply:
Resolved with ed6c261